### PR TITLE
WT-11834 Add total time to IO and operation statistics

### PR DESCRIPTION
**Pull request by y123456yz**
Background:
1.  wt does not have average time consuming statistics. Adding total time consuming statistics can help us obtain average time consuming of nodes. Average time consuming =perf_hist_opwrite_latency_total/(perf_hist_opwrite_latency_lt100+perf_hist_opwrite_latency_lt250+ perf_HIST_opwrite_opency_LTWrite_ latency_lt500+perf_hist_opwrite_latency_lt1000+perf_hist_opwrite_latency_lt10000+perf_hist_opwrite_latency_gt10000)/ (Sampling period).
2.  Sometimes we often encounter node delay jitter, and we do not know whether it is caused by mongo server or wiredtiger. Increasing the statistics can help quickly locate the problem caused by who
3. by using (perf_hist_opwrite_latency_lt100+perf_hist_opwrite_latency_lt250+perf_hist_opwrite_latency_lt500+perf_hist_opwrite_laten cy_lt1000+perf_hist_opwrite_latency_lt10000+perf_hist_opwrite_latency_gt10000)/( Sampling period ) , the read/write qps of nodes can be obtained, which can help us obtain qps statistics.

### CODE REVIEW

**Comment by y123456yz** [comment]
https://jira.mongodb.org/browse/WT-11834

**Comment by korteland** [comment]
Hi @y123456yz ! Thanks for the PR, I plan to review this within the next few days.

**Comment by korteland** [comment]
Before I review this - would you mind running `s_all` please? I can see that it's failed with these changes.

**Comment by y123456yz** [comment]
> Before I review this - would you mind running `s_all` please? I can see that it's failed with these changes.

I have run the s_all，and fix the s_stat problem
thanks.

File: src/include/stat.h
Comment (korteland) [inline src/include/stat.h:274]: This comment is no longer correct - would you be able to remove the last two sentences from this paragraph, please?

(I'm not too concerned about the mentioned problem of excessively large smallest values - if it becomes a problem, we can revert the change.)

**Review by korteland — COMMENTED** [review]
One comment update - LGTM otherwise. Thanks for the contribution!

File: src/include/stat.h
Comment (y123456yz) [reply src/include/stat.h:274]: hi, @korteland 
I have delete the no longer correct comment.

I have calculated that it will not exceed the limit, assuming that each request takes 1000us and 100,000 requests per second, this parameter can be used for 5,849 years(18446744073709551615/1000/10000/3600/24/365=5849years).

thanks

Comment (korteland) [reply src/include/stat.h]: Thanks @y123456yz ! This looks good to me now.

**Review by korteland — APPROVED** [review]
LGTM - once you resolve the merge conflicts I'll go ahead and merge this in. Thanks for improving these stats!

**Comment by y123456yz** [comment]
> LGTM - once you resolve the merge conflicts I'll go ahead and merge this in. Thanks for improving these stats!

hi, @korteland 

because there was a lot of conflicting code, I regenerated the file with dist/s_all with the newest develop branch code, Please help to check if there is any problem.

thanks

**Comment by korteland** [comment]
Hi @y123456yz - no issues, just waiting for the testing to run its course and I'll merge this in :-)

**Comment by korteland** [comment]
evergreen merge

**Pull request merged**
